adoc/Private-Registry-Installation.adoc

```diff
@@ -402,7 +408,7 @@ The above settings will be ignored and may be omitted for components configured
 
 [WARNING]
 ====
-If a {kube} persistent volume is configured to store OCI artifacts instead of an external storage service, and if your {kube} cluster does not provide a `StorageClass` with `ReadWriteMany` access mode capabilities, the `updateStrategy.type` option must set to `Recreate` in the `harbor-values.yaml` file, otherwise running `helm upgrade` to apply subsequent configuration changes or to perform upgrades will result in failure:
+If a {kube} persistent volume is configured to store OCI artifacts instead of an external storage service, and if your {kube} cluster does not provide a `StorageClass` with `ReadWriteMany` access mode capabilities, the `updateStrategy.type` option must set to `Recreate` in the `harbor-values.yaml` file; otherwise, running `helm upgrade` to apply subsequent configuration changes or to perform upgrades will result in failure:
```

Comment:
Is there a typo?? 

...apabilities, the `updateStrategy.type` option must be set to `Recreate` in the `harbor-values.yaml` file; ...

The entire sentence is quite long. Instead of the semicolon before "otherwise", can there be used a period?

Reply:
Good point. Broken into 2 sentences at that point.